java/connector-node/risingwave-sink-jdbc/src/main/java/com/risingwave/connector/JDBCSinkConfig.java

```diff
@@ -34,15 +36,35 @@ public class JDBCSinkConfig extends CommonSinkConfig {
 
     @JsonCreator
     public JDBCSinkConfig(
-            @JsonProperty(value = "jdbc.url") String jdbcUrl,
-            @JsonProperty(value = "table.name") String tableName,
-            @JsonProperty(value = "type") String sinkType) {
+            @JsonProperty(value = "jdbc.url", required = true) String jdbcUrl,
```

Comment:
Previously we put the required fields in the constructor to indicate that the field is required, and it does report an error when the fields is not specified in the map. 

Therefore, I am not sure what is the expected behavior of the current usage. Does it require the user to explicitly set the `schema.name` property as null when they don't have a schema to specify, or users can just don't set it?

Reply:
Since `schema.name` is optional, so we should not throw error if user doesn't provide it in the WITH options. But previous implementation `mapper.configure(DeserializationFeature.FAIL_ON_MISSING_CREATOR_PROPERTIES, true)` will throw error if `schema.name` is absent.
So I remove the configuration and configures required field in the constructor instead. User doesn't need to provide `schema.name` if they don't need it. For example the mysql-cdc case.

Reply:
I think we should avoiding modifying `mapper.configure(DeserializationFeature.FAIL_ON_MISSING_CREATOR_PROPERTIES, true)` in this PR, since it will cause user facing change in other sink. If we want to make the `schema.name` optional, you can avoiding passing it in the constructor, and instead mark its field with the annotation. This is our current convention to handle optional field.

Reply:
Cool, good to know we have a convention for that. :)